modules/form-field/react/spec/Hint.spec.tsx

```diff
@@ -1,28 +1,24 @@
 import * as React from 'react';
-import {shallow, mount} from 'enzyme';
+import {render} from '@testing-library/react';
 import Hint from '../lib/Hint';
 
 describe('Hint', () => {
-  test('Error label is set', () => {
-    const component = shallow(<Hint error={Hint.ErrorType.Error} />);
-
-    expect(component.render().text()).toEqual(expect.stringContaining('Error:'));
-
-    component.unmount();
+  describe('when rendered with an ErrorType', () => {
+    it('should render an error label when error type is Error', async () => {
+      const {getByTestId} = render(<Hint data-testid="hintError" error={Hint.ErrorType.Error} />);
+      expect(getByTestId('hintError').children[0].textContent).toEqual('Error: ');
+    });
+    it('should render an alert label when error type is Alert', async () => {
+      const {getByTestId} = render(<Hint data-testid="hintError" error={Hint.ErrorType.Alert} />);
+      expect(getByTestId('hintError').children[0].textContent).toEqual('Alert: ');
+    });
   });
-
-  test('Alert label is set', () => {
-    const component = shallow(<Hint error={Hint.ErrorType.Alert} />);
-
-    expect(component.render().text()).toEqual(expect.stringContaining('Alert:'));
-
-    component.unmount();
-  });
-
-  test('Hint should spread extra props', () => {
-    const component = mount(<Hint data-propspread="test" />);
-    const container = component.at(0).getDOMNode();
-    expect(container.getAttribute('data-propspread')).toBe('test');
-    component.unmount();
+  describe('when rendered with a custom Error string', () => {
+    it('should render a custom label for type Error', async () => {
+      const {getByTestId} = render(
+        <Hint errorLabel={'Hay un Error'} data-testid="hintError" error={Hint.ErrorType.Error} />
+      );
+      expect(getByTestId('hintError').children[0].textContent).toEqual('Hay un Error: ');
```

Comment:
I'm not sure this is the best way to get the text, I'm open to improving this! @anicholls @NicholasBoll

Reply:
Rather than finding the div and then checking its text content, I would just use `getByText`!